newIDE/app/src/VariablesList/MultilineVariableEditorDialog.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+import Dialog, { DialogPrimaryButton } from '../UI/Dialog';
+import SemiControlledTextField from '../UI/SemiControlledTextField';
+
+type Props = {|
+  initialValue: string,
+  onClose: (newValue: string) => void,
+|};
+
+export const MultilineVariableEditorDialog = ({
+  initialValue,
+  onClose,
+}: Props) => {
+  const [value, setValue] = React.useState(initialValue);
+
+  return (
+    <Dialog
+      open
+      title={null}
+      flexColumnBody
+      actions={[
+        <DialogPrimaryButton
+          id="ok"
+          label={<Trans>Ok</Trans>}
+          primary
+          onClick={() => onClose(value)}
+        />,
+      ]}
+      maxWidth="md"
```

Comment:
I wonder if on mobile the dialog should not take the full screen

Reply:
Indeed, I'm adding the prop to prevent fullscreen on small screens